src/core/StaticIcon/__snapshots__/StaticIcon.test.tsx.snap

```diff
@@ -6,18 +6,31 @@ exports[`calling render with the same component on the same container does not r
   vertical-align: baseline;
 }
 
+<<<<<<< HEAD
```

Comment:
This snapshot seem to have some Git stuff going 👀

Reply:
Fixed already. Bad rebase.